### PR TITLE
Fix Duplicate loading animation

### PR DESCRIPTION
**Pull request by grafixeyehero**
**Issues**
Duplicate loading animation #199

### CODE REVIEW

File: src/components/htmlvideoplayer/plugin.js [hunk 1]
```diff
@@ -1398,12 +1398,7 @@ define(['browser', 'require', 'events', 'apphost', 'loading', 'dom', 'playbackMa
                             cssClass += ' htmlvideoplayer-moveupsubtitles';
                         }
 
-                        // Can't autoplay in these browsers so we need to use the full controls, at least until playback starts
                         if (!appHost.supports('htmlvideoautoplay')) {
-                            html += '<video class="' + cssClass + '" preload="metadata" autoplay="autoplay" controls="controls" webkit-playsinline playsinline>';
-                        } else {
-
-                            // Chrome 35 won't play with preload none
                             html += '<video class="' + cssClass + '" preload="metadata" autoplay="autoplay" webkit-playsinline playsinline>';
```
Comment (JustAMan) [inline src/components/htmlvideoplayer/plugin.js]: What about browsers which do have autoplay? Won't they now have no video player at all?

**Comment by grafixeyehero** [comment]
What did you mean no video player at all?

**Comment by grafixeyehero** [comment]
Wrong btn sorry

**Comment by grafixeyehero** [comment]
Tested
Chrome 76.0.3809.132
Firefox 70.0b6

**Comment by dkanada** [comment]
This adds a duplicate loading animation for firefox on my machine. I definitely want to merge this once we figure out the behavior but this doesn't seem like the right solution.

**Comment by grafixeyehero** [comment]
> 
> 
> This adds a duplicate loading animation for firefox on my machine. I definitely want to merge this once we figure out the behavior but this doesn't seem like the right solution.

the first comment did'nt hide on firefox. i hope this fix https://github.com/jellyfin/jellyfin-web/pull/468/commits/20c23dc7285b831bdf130275c1e6aa760a7282dc

**Comment by grafixeyehero** [comment]
@dkanada could you explain me a in which version you test and the problem that you faced. is it the controller or the loading animation

**Pull request merged**